package.json

```diff
@@ -20,6 +20,7 @@
     "@babel/preset-env": "^7.11.5",
     "@babel/register": "^7.11.5",
     "@babel/runtime": "^7.11.2",
+    "babel-eslint": "^10.1.0",
```

Comment:
why babel eslint? what is its function?

Reply:
> why babel eslint? what is its function?

Eslint was giving a `parsing error` message on codeclimate for the arrow function in one of the files. I found this [solution](https://stackoverflow.com/a/43426331/11990570). The error went away after I implemented it.